app/javascript/styles/components.scss

```diff
@@ -474,15 +474,18 @@
   }
 }
 
-a.status__content__spoiler-link {
```

Comment:
does this refer to links within the spoiler text or the 'link' used to show/hide the status content?

if the latter, should this still be called blabla-link or should it be blabla-button since you changed it from a link to a button?